microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -0,0 +1,1284 @@
+* [CoreWebView2Environment.GetAutomationProviderForWindow Method] TODO: no C# link, ok?
+-->
+
+##### [C++](#tab/cpp)
+
+* [ICoreWebView2CompositionController2::AutomationProvider property (get)](https://docs.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller2#get_automationprovider)<!--no put-->
+* [ICoreWebView2Environment4::GetAutomationProviderForWindow method](https://docs.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2environment4#getautomationproviderforwindow)<!--not in c#-->
+
+---
+
+
+<!-- ====================================================================== -->
+## User data
+
+Manage the user data folder (UDF), which is a folder stored on the user's machine, that contains data related to the host app and WebView2.  WebView2 apps use user data folders to store browser data, such as cookies, permissions, and cached resources.
+
+See also:
+* [Manage user data folders](user-data-folder.md)
+
+##### [C#](#tab/c-sharp)
+
+* [CoreWebView2Environment.UserDataFolder Property](https://docs.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2environment.userdatafolder)
+* [CoreWebView2EnvironmentOptions.ExclusiveUserDataFolderAccess Property](https://docs.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.exclusiveuserdatafolderaccess)<!--new-->
+* [CoreWebView2Profile Class](https://docs.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2profile)<!--new-->
+   * `ClearBrowsingDataAsync` Methods
+* [CoreWebView2Environment Class](https://docs.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2environment)<!--new-->
+   * [CoreWebView2Environment.UserDataFolder Property](https://docs.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2environment.userdatafolder)
+* [CoreWebView2Environment interface](https://docs.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2environment)<!--new--><!--TODO: why is this link here, which members?-->
+
+##### [C++](#tab/cpp)
+
+* [ICoreWebView2Environment7::UserDataFolder property (get)](https://docs.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2environment7#get_userdatafolder)<!--no put-->
+* [ICoreWebView2EnvironmentOptions2::ExclusiveUserDataFolderAccess property (get](https://docs.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions2#get_exclusiveuserdatafolderaccess), [put)](https://docs.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions2#put_exclusiveuserdatafolderaccess)<!--new-->
+* [ICoreWebView2Profile2 interface](https://docs.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2profile2)<!--new-->
+   * `ClearBrowsingData` methods
+* [ICoreWebView2Environment10 interface](https://docs.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2environment10)<!--new--><!--TODO: why is this link here, which members?-->
```

Comment:
@nishitha-burman  - why link to ICoreWebView2Environment10?  similar q for C# above

Reply:
closing "outdated" version of this cmt